growattServer/__init__.py

```diff
@@ -22,9 +22,11 @@ class Timespan(IntEnum):
     hour = 0
     day = 1
     month = 2
+    year = 3
+    all = 4
 
 class GrowattApi:
-    server_url = 'https://server-api.growatt.com/'
+    server_url = 'https://server.growatt.com/'
```

Comment:
server-api.growatt.com is still the URL being used by the app, as not everyone is having issues with the current URL and the new URL might respond differently.
It could cause issues in currently running code.
It's best to overwrite this URL in the scripts using it
```suggestion
    server_url = 'https://server-api.growatt.com/'
```

Reply:
This is outdated, see https://github.com/indykoning/PyPi_GrowattServer/pull/77